playbooks/install_os.yml

```diff
@@ -17,10 +28,25 @@
         tasks_from: install.yml
       tags:
         - edpm_sshd
-    - name: Install edpm_chrony
-      ansible.builtin.import_role:
-        name: osp.edpm.edpm_chrony
-        tasks_from: install.yml
+    - name: Set FQCN for the timesync role
+      ansible.builtin.set_fact:
+        timesyncfqn:
+          "{%- if ansible_facts['distribution'] == 'RedHat' -%}
+            redhat.rhel{%- else -%}
+            fedora.linux{%- endif -%}
+          _system_roles.timesync"
+      delegate_to: localhost
+    - name: Compile list of ntp servers with their attributes
+      ansible.builtin.set_fact:
+        timesync_ntp_servers: "{{ timesync_ntp_servers | default([]) + [{ 'hostname' : item }]}}"
+      loop: "{{ timesync_ntp_servers }}"
+      when: item is string
+      failed_when: >
+        ( item is mapping ) and
+        ( 'hostname' not in item )
+    - name: Install and configure time service using timesync system role
```

Comment:
So you're building this list in lines 40-46 so that the patches which depend on this one can keep passing a list of NTP servers instead of passing a dictionary which maps hostname to NTP server as pointed out here.

  https://github.com/openstack-k8s-operators/dataplane-operator/pull/463/files#r1355074751
  https://github.com/openstack-k8s-operators/install_yamls/pull/575/files#r1356693584

Reply:
Pretty much yes. It does minimize the number of things that can break. The existing usage stays almost the same, all important changes will be contained here.

Reply:
List compilation and checks removed.